src/main/kotlin/io/ionic/libs/ionfileviewerlib/helpers/IONFLVWOpenDocumentHelper.kt

```diff
@@ -0,0 +1,74 @@
+package io.ionic.libs.ionfileviewerlib.helpers
+
+import android.app.Activity
+import android.content.ActivityNotFoundException
+import android.content.Intent
+import android.net.Uri
+import android.webkit.MimeTypeMap
+import androidx.core.content.FileProvider
+import java.io.File
+import java.io.FileNotFoundException
+
+internal class IONFLVWOpenDocumentHelper {
+    private val Activity.contentProviderAuthority: String
+        get() = "$packageName.opener.provider"
+
+    /**
+     * Opens a file from a local file path
+     *
+     * @param activity the Android activity
+     * @param filePath the full path to the file
+     * @throws ActivityNotFoundException if no activity/app was found to open the file
+     * @throws FileNotFoundException if the file does not exist
+     */
+    @Throws(ActivityNotFoundException::class, FileNotFoundException::class)
+    fun openDocumentFromLocalPath(activity: Activity, filePath: String) {
+        val mimeType = getMimeType(filePath)
+        val file = File(filePath.replace("file:///", ""))
+
+        if (file.exists()) {
```

Comment:
This is a matter of "taste" I'm sure but we could use `if (!file.exists())` and return early, instead of using if-else.